pkg/noderesourcetopology/pluginhelpers.go

```diff
@@ -64,6 +69,14 @@ func initNodeTopologyInformer(tcfg *apiconfig.NodeResourceTopologyMatchArgs, han
 		return nil, err
 	}
 
+	// TODO: do we have a better name to introspect the profile name?
+	if schedName, ok := os.LookupEnv(schedulerNameEnvVar); ok {
```

Comment:
This requires an update of both docs and manifests

Reply:
I think it's better to just postpone this change. I'll remove the related commit from the next upload

Reply:
this still needs to be addressed

Reply:
in general, need to add a commit to document the new option into the README. Will fix in the next upload.

Reply:
hrm, one scheduler binary can contain multiple profiles that the NRT plugin is enabled in. Let me think about it.

Reply:
Not sure if it works: enforce cast `handle` to `framework.Framework` in plugin's initialization, and call its `ProfileName()` to get the profile name and store it in the plugin. (but even if it works, it doesn't quite fit the case that NRT is enabled in multiple profiles unless we put a shared variable to store the profile names)

Reply:
addressed the comment docs and manifests. Thanks for pointing this out @Tal-or !

Reply:
> We can pivot to a (lock-protected) module-level sets.String if you prefer.

It can be lock-free as each plugin's New() is called sequentially in the scheduler framework core.

Reply:
very cool, thanks. Moving to this approach. UX is much more streamlined.